src/components/prompt-text.ts

```diff
@@ -217,10 +207,32 @@ export class PromptText extends EventManager {
                 }
                 case "Word": {
                     if (this.levelData.levelMeta.protoType == "Visible") {
+                    if(this.targetStones.length!=this.currentPromptText.length){
+                    if(this.targetStones.length>i){   
+                    if (this.droppedStoneCount > i || this.droppedStoneCount== undefined) {
+                        this.context.fillStyle = "black";
```

Comment:
Code can be refactored and handled easily

this.context.fillStyle = (this.droppedStoneCount > i || this.droppedStoneCount === undefined) ? "black" : "red";

Reply:
Thanks @ashwinnair-chimple  for you feedback. I have made all the necessary  changes requested by you🫡.